test/unit/sync/branch.test.ts

```diff
@@ -233,7 +233,7 @@ describe.skip("sync/branches", () => {
 				add: jest.fn()
 			}
 		};
-		await expect(processInstallation(app, queues)(job, getLogger('test'))).toResolve();
+		await expect(processInstallation(app, queues)(job, getLogger("test"))).toResolve();
```

Comment:
Can we avoid changing format everywhere and doing changes in the same PR?
Makes it extremely difficult to read